VideoLocker/src/main/java/org/edx/mobile/model/course/CourseStructureJsonHandler.java

```diff
@@ -59,29 +57,26 @@ public BlockData deserialize(JsonElement json, Type typeOfT,
         }
     }
 
-    class BlockListDeserializer implements JsonDeserializer<BlockList>
-    {
+    private static class BlockListDeserializer implements JsonDeserializer<BlockList> {
         @Override
-        public BlockList deserialize(JsonElement json, Type typeOfT, JsonDeserializationContext context) throws JsonParseException
-        {
-            Type mapType = new TypeToken<Map<String, BlockModel>>() {}.getType();
+        public BlockList deserialize(JsonElement json, Type typeOfT, JsonDeserializationContext context) throws JsonParseException {
+            Type mapType = new TypeToken<Map<String, BlockModel>>() {
+            }.getType();
```

Comment:
It was probably done by the auto-formatter, which can be a bit retarded at times :confounded: